presentation/src/main/java/me/androidbox/presentation/event/viewmodel/EventViewModel.kt

```diff
@@ -294,44 +296,53 @@ class EventViewModel @Inject constructor(
         }
     }
 
-    private fun insertEventDetails(eventId: String) {
+    private fun insertEventDetails() {
         val startDateTime = AlarmReminderProvider.getCombinedDateTime(eventScreenState.value.startTime, eventScreenState.value.startDate)
         val endDateTime = AlarmReminderProvider.getCombinedDateTime(eventScreenState.value.endTime, eventScreenState.value.endDate)
         val remindAt = AlarmReminderProvider.getRemindAt(eventScreenState.value.alarmReminderItem, startDateTime)
 
-        val event = Event(
-            id = eventId,
-            title = eventScreenState.value.eventTitle,
-            description = eventScreenState.value.eventDescription,
-            startDateTime = startDateTime.toEpochSecond(),
-            endDateTime = endDateTime.toEpochSecond(),
-            remindAt = remindAt.toEpochSecond(),
-            eventCreatorId = preferenceRepository.retrieveCurrentUserOrNull()?.userId ?: "",
-            isUserEventCreator = true,
-            isGoing = true,
-            attendees = eventScreenState.value.attendees,
-            photos = eventScreenState.value.listOfPhotoUri
-        )
+        preferenceRepository.retrieveCurrentUserOrNull()?.userId?.let { userId ->
```

Comment:
I am just wondering what to do if the userId was null. As its a nullable type is it worth checking if its null? Is it possible that it could be null. And in cases of null what is the best way to handle.
In my case I check for null and if its null show the user a snackbar or toast message saying that insert cannot proceed? Is this correct?

Reply:
Would this ever be the case here? If so, it would rather be a bug and not a situation which the user can really cause. Usually apps just show a generic error message in this case such as `Oops, something went wrong!`